<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lacommande</class>
 <widget class="QMainWindow" name="lacommande">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1554</width>
    <height>958</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_searchWord">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rechercher">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>RECHERCHE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_navigate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>PRODUIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rapport">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>650</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>RAPPORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ExportPDF">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>650</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_employe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>EMPLOYE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ajouter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>600</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>AJOUTER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stats">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>430</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>STATISTIQUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>mode_paiement</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_statut">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supprimer">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>600</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>SUPPRIMER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>statue</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rechercheID">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modifier">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>600</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>EDITER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serviceButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>SERVICE</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>995</x>
      <y>170</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_commande">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>COMMANDE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CheckEnCours">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>600</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>ENVOYER</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>530</y>
      <width>104</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_trier">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>TRI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ModePaiement">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>540</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelqgr">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>3301</width>
      <height>877</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="background.qrc">:/img/backgroundIMG/Untitled design (2).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>labelqgr</zorder>
   <zorder>lineEdit_searchWord</zorder>
   <zorder>pushButton_rechercher</zorder>
   <zorder>pushButton_navigate</zorder>
   <zorder>pushButton_rapport</zorder>
   <zorder>pushButton_ExportPDF</zorder>
   <zorder>pushButton_employe</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_ajouter</zorder>
   <zorder>lineEdit_ID</zorder>
   <zorder>pushButton_Stats</zorder>
   <zorder>label_6</zorder>
   <zorder>lineEdit_statut</zorder>
   <zorder>lineEdit_date</zorder>
   <zorder>pushButton_supprimer</zorder>
   <zorder>label_4</zorder>
   <zorder>tableView</zorder>
   <zorder>lineEdit_rechercheID</zorder>
   <zorder>pushButton_modifier</zorder>
   <zorder>label_3</zorder>
   <zorder>serviceButton</zorder>
   <zorder>graphicsView</zorder>
   <zorder>pb_commande</zorder>
   <zorder>pushButton_CheckEnCours</zorder>
   <zorder>textEdit_log</zorder>
   <zorder>pushButton_trier</zorder>
   <zorder>lineEdit_ModePaiement</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1554</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
